src/components/ContentHighlights/ContentHighlightCardItem.jsx

```diff
@@ -1,36 +1,49 @@
 import React from 'react';
-import { Card } from '@edx/paragon';
+import { Card, Hyperlink } from '@edx/paragon';
 import Truncate from 'react-truncate';
 import PropTypes from 'prop-types';
-import { FOOTER_TEXT_BY_CONTENT_TYPE } from './data/constants';
+import { getContentHighlightCardFooter } from './data/constants';
 
 const ContentHighlightCardItem = ({
+  isLoading,
   title,
+  hyperlink,
   contentType,
   partners,
   cardImageUrl,
+  price,
 }) => {
-  const cardLogoSrc = partners?.length === 1 ? partners[0].logoImageUrl : undefined;
-  const cardLogoAlt = partners?.length === 1 ? `${partners[0].name}'s logo` : undefined;
-  const cardSubtitle = partners?.map(p => p.name).join(', ');
-
+  const cardInfo = {
+    cardTitle: (<Truncate lines={3} title={title}>{title}</Truncate>),
+    cardLogoAlt: partners?.length === 1 ? `${partners[0].name}'s logo` : undefined,
+    cardLogoSrc: partners?.length === 1 ? partners[0].logoImageUrl : undefined,
+    cardSubtitle: partners?.map(p => p.name).join(', '),
+    cardFooter: getContentHighlightCardFooter(price, contentType),
+  };
+  if (hyperlink) {
+    cardInfo.cardTitle = (
+      <Hyperlink destination={hyperlink} target="_blank">
+        <Truncate lines={3} title={title}>{title}</Truncate>
+      </Hyperlink>
+    );
+  }
   return (
-    <Card>
+    <Card isLoading={isLoading}>
       <Card.ImageCap
         src={cardImageUrl}
         srcAlt=""
-        logoSrc={cardLogoSrc}
-        logoAlt={cardLogoAlt}
+        logoSrc={cardInfo.cardLogoSrc}
+        logoAlt={cardInfo.cardLogoAlt}
       />
       <Card.Header
-        title={<Truncate lines={3} title={title}>{title}</Truncate>}
-        subtitle={<Truncate lines={2} title={cardSubtitle}>{cardSubtitle}</Truncate>}
+        title={cardInfo.cardTitle}
+        subtitle={<Truncate lines={2} title={cardInfo.cardSubtitle}>{cardInfo.cardSubtitle}</Truncate>}
```

Comment:
[curious] Why is the `Truncate` component part of `cardInfo.cardTitle` but not `cardInfo.cardSubtitle`? I feel these should be consistent. E.g., maybe the variable stored in `cardInfo.cardTitle` is only the raw text and the `Truncate` component is rendered within the `title` prop itself, similar to how `subtitle` is currently set up.

Reply:
I was debating leaving `cardInfo` for just data with altered logic or whether to include all card info.  I'll take this as confirmation that the `subtitle` also needs to be pulled into the `cardInfo` object

Reply:
Going back to this again, i remembered why I structured it as so. In creating this `ContentHighlightCardItem`, I went with the mindset that not every component will be used if we hope to make the card component extensible in the future. So defining the title in the `Card.Header` as an object was done purposely for the `hyperlink` (it may or may not exist, giving users the object), while not defining the subtitle in the object due to its static nature in the `Card.Header` (no logic needed to define what goes in it beyond the `partners`.